.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: objective-c
-osx_image: xcode7
+osx_image: xcode8
```

Comment:
do they not have an 8.1 image yet? Or is this purposely 8 so that it uses 3.0 instead of 3.0.1?